community-edition/readme.md

```diff
@@ -54,62 +63,54 @@ gcloud auth login
 `sudo apt update && sudo apt install npm && sudo npm install pem-jwk -g`
 ### install k3s
 ```
-### install k3s without traefik
+
+### Step 2: Install k3s without traefik
```

Comment:
yeah i see the structure of the 2 examples there are kind of messed up, i'll get it fixed shortly